frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -14,18 +21,41 @@ import {
 
 const baseClass = "app-config-form";
 
+type IAdvancedConfigFormData = Pick<
```

Comment:
Thanks for spearheading cleaning up the interfaces on these pages. Why do we need this huge `IConfigFormData` interface at all if we're just picking it apart for each page? Why not just have a a `FormData` interface for each page?

Reply:
gonna switch now to smaller interfaces, pr incoming in an hour